config/helm/crd_controlplane_webhook.yaml

```diff
@@ -0,0 +1,22 @@
+# The following patch enables conversion webhook for CRD
```

Comment:
Patch for setting our conversion spec (overriding namespace part) in all controlplane CRDs (label selector is used in kustomization.yaml to apply this to all controlplane CRDs).